### PR TITLE
Add TravisCI script for RC_1_0 branch

### PR DESCRIPTION
**Pull request by Chocobo1**
Recent update in RC_1_0 branch broke our TravisCI compilation, so I notice there wasn't a TravisCI script in RC_1_0 branch.

### CODE REVIEW

**Comment by Chocobo1** [comment]
Please don't pull this now, still trying to fix errors on TravisCI.

**Comment by Chocobo1** [comment]
However using `./autotool.sh && ./configure && make` works reliably. Use that instead?

File: Jamfile [hunk 1]
```diff
@@ -468,21 +469,23 @@ feature export-extra : off on : composite propagated ;
 
 # this is a trick to get filename paths to targets to become shorter
 # making it possible to build on windows, especially mingw seems particular
+# for release builds, disable optimizations as they bump GCC over the edge of
+# allowed memory usage on travis-ci
 variant test_release
 	: release : <asserts>production <debug-symbols>on
-	  <invariant-checks>full <boost-link>shared
-	  <export-extra>on <debug-iterators>on <boost>source <threading>multi
+	  <invariant-checks>full <boost-link>shared <optimization>off
```
Comment (arvidn) [inline Jamfile]: I would like to have one build configuration that makes sure release builds still work and pass the tests. why did you have to disable optimizations?

Comment (arvidn) [inline Jamfile]: oh, right. I missed the comment

**Pull request merged**